projects/canopy/src/lib/hero/hero-header/hero-header.component.scss

```diff
@@ -10,7 +10,7 @@
   bottom: 0;
   left: 0;
   right: 0;
-  background-color: var(--hero-bg-color);
+  background-color: var(--hero-overlay-bg-color);
   opacity: var(--hero-bg-alpha);
```

Comment:
Doesn't that mean that any content inside the hero is also going to be slightly opaque? If so, wouldn't an rgba background colour be better?

Reply:
Ignore me, its the overlay isn't it